src/typescript-reporter/reporter/TypeScriptConfigurationParser.ts

```diff
@@ -114,10 +114,10 @@ function removeJsonExtension(path: string) {
   }
 }
 
-function getTsBuildInfoEmitOutputFilePath(options: ts.CompilerOptions) {
-  if (typeof ts.getTsBuildInfoEmitOutputFilePath === 'function') {
+function getTsBuildInfoEmitOutputFilePath(typescript: typeof ts, options: ts.CompilerOptions) {
+  if (typeof typescript.getTsBuildInfoEmitOutputFilePath === 'function') {
```

Comment:
hm, there does seem to be some issue with getTsBuildInfoEmitOutputFilePath causing failures when using the passed in version. Seems something may be swallowing it, as this is how the issue is surfacing

```
Error: No handler found for procedure getReport.
Error: No handler found for procedure getReport.
    at Object.<anonymous> (/tmp/fork-ts-checker-sandbox-7rVqZk/node_modules/fork-ts-checker-webpack-plugin/lib/rpc/RpcService.js:21:27)
    at Generator.next (<anonymous>)
    at /tmp/fork-ts-checker-sandbox-7rVqZk/node_modules/fork-ts-checker-webpack-plugin/lib/rpc/RpcService.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/tmp/fork-ts-checker-sandbox-7rVqZk/node_modules/fork-ts-checker-webpack-plugin/lib/rpc/RpcService.js:4:12)
    at callListener (/tmp/fork-ts-checker-sandbox-7rVqZk/node_modules/fork-ts-checker-webpack-plugin/lib/rpc/RpcService.js:16:39)
    at messageListeners.forEach (/tmp/fork-ts-checker-sandbox-7rVqZk/node_modules/fork-ts-checker-webpack-plugin/lib/rpc/rpc-ipc/RpcIpcMessagePort.js:31:13)
    at Set.forEach (<anonymous>)
    at process.handleMessage (/tmp/fork-ts-checker-sandbox-7rVqZk/node_modules/fork-ts-checker-webpack-plugin/lib/rpc/rpc-ipc/RpcIpcMessagePort.js:30:26)
    at process.emit (events.js:198:13)
```

Any thoughts @piotr-oles ?

Reply:
It almost seems like just how things are being imported is causing the memory issue to be hit a little sooner (before RPC handlers are set up)

Reply:
It seems like another bug that has been revealed by this change - we should close the reporter on an unsuccessful `configure` call. I will send a fix